Sources/SnapshotTesting/Strategy/NSImage.swift

```diff
@@ -117,4 +100,23 @@ private func context(for cgImage: CGImage) -> CGContext? {
   context.draw(cgImage, in: CGRect(x: 0, y: 0, width: cgImage.width, height: cgImage.height))
   return context
 }
+
+private func diff(_ old: NSImage, _ new: NSImage) -> NSImage {
+  let oldCiImage = CIImage(cgImage: old.cgImage(forProposedRect: nil, context: nil, hints: nil)!)
```

Comment:
Using CoreImage now for diffing. It produces something very close to what we see in Kaleidoscope.